server/src/testFixtures/java/io/crate/operation/aggregation/AggregationTestCase.java

```diff
@@ -234,12 +236,7 @@ public Object executeAggregation(Signature maybeUnboundSignature,
                     is(partialResultWithDocValues.get(0).get(0))
                 );
             } else {
-                var docValueAggregator = aggregationFunction.getDocValueAggregator(
-                    actualArgumentTypes,
-                    Lists2.map(actualArgumentTypes, dataType -> mock(MappedFieldType.class)),
-                    List.of()
-                );
-                if (docValueAggregator != null) {
+                if (getDocValueAggregator(maybeUnboundSignature, actualArgumentTypes, actualReturnType, mock(DocTableInfo.class)) != null) {
```

Comment:
Why wasn't it possible to keep calling `aggregationFunction.getDocValueAggregator` with the changed arguments? 

Calling `DocValuesAggregates.createAggregators` means that the test-cases now use more mocked/test code instead of more of the regular production code paths.

Reply:
I had false assumption that `createAggregators` could be treated equal to `getDocValueAggregator`.
Reverted almost everything back to as it was before for `AggregationTestCase` and added minimal changes to adapt my work.